mirgecom/initializers.py

```diff
@@ -39,9 +39,104 @@
 .. autoclass:: SodShock1D
 .. autoclass:: Lump
 .. autoclass:: Uniform
+.. autoclass:: AcousticPulse
+.. automethod: _make_pulse
+.. automethod: _set_uniform_solution
 """
 
 
+def _set_uniform_solution(x_vec, mass=1.0, energy=2.5, pressure=1.0,
```

Comment:
Why "set"? What's being set? It looks like this *returns* a uniform solution.

Reply:
Fixed